AmplifyPlugins/Storage/Tests/StorageHostApp/copy_configuration.sh

```diff
@@ -3,7 +3,14 @@
 set -e
 
 SOURCE_DIR=$HOME/.aws-amplify/amplify-ios/testconfiguration
-DESTINATION_DIR="${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}/testconfiguration/"
+DESTINATION_DIR="$SOURCE_ROOT"
+
+if [ -d "$AMPLIFY_CONFIGURATION_PATH" ]; then
+    echo "Found AMPLIFY_CONFIGURATION_PATH - copying"
+    mkdir -p "$DESTINATION_DIR"
+    ditto "$AMPLIFY_CONFIGURATION_PATH" "$DESTINATION_DIR"
+    exit 0
+fi
```

Comment:
This whole $AMPLIFY_CONFIGURATION_PATH block will be used by the new CICD system since writing to $HOME is not allowed.